packages/contracts-bedrock/src/Safe/LivenessModule.sol

```diff
@@ -0,0 +1,183 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Safe, OwnerManager } from "safe-contracts/Safe.sol";
+import { Enum } from "safe-contracts/common/Enum.sol";
+import { OwnerManager } from "safe-contracts/base/OwnerManager.sol";
+import { LivenessGuard } from "src/Safe/LivenessGuard.sol";
+import { ISemver } from "src/universal/ISemver.sol";
+
+// TODO(maurelian): remove me
+import { console2 as console } from "forge-std/console2.sol";
+
+/// @title LivenessModule
+/// @notice This module is intended to be used in conjunction with the LivenessGuard. It should be able to
+///         execute a transaction on the Safe in only a small number of cases.
+contract LivenessModule is ISemver {
+    /// @notice The Safe contract instance
+    Safe public immutable safe;
```

Comment:
The immutables do not follow the screaming snake case pattern. Would need to make them internal and then add getters with the correct casing

Reply:
90c826e5f66a7d4452d27664ba438cda139518c6